src/containers/Tasks/AttestClaim/AttestClaim.tsx

```diff
@@ -59,15 +69,29 @@ class AttestClaim extends React.Component<Props, State> {
 
   public render(): JSX.Element {
     const { requestForAttestation } = this.props
+    const { quote } = this.state
     return (
       <section className="AttestClaim">
         <ClaimDetailView claim={requestForAttestation.claim} />
 
         <AttestedClaimsListView
           attestedClaims={requestForAttestation.legitimations}
           delegationId={requestForAttestation.delegationId}
-          context="legitimations"
+          context="terms"
         />
+        {!quote ? (
```

Comment:
why the inverse order? may be a little confusing

Reply:
Agreed, I switched it around. 

Also, the styling just sucks here :)

Reply:
Also switched it in other places.